src/include/ceph_features.h

```diff
@@ -110,6 +110,7 @@ DEFINE_CEPH_FEATURE_RETIRED(19, 1, CHUNKY_SCRUB, JEWEL, LUMINOUS)
 DEFINE_CEPH_FEATURE(19, 2, OSD_PGLOG_HARDLIMIT)
 
 DEFINE_CEPH_FEATURE_RETIRED(20, 1, MON_NULLROUTE, JEWEL, LUMINOUS)
+DEFINE_CEPH_FEATURE(20, 2, OSD_COPY_FILE) // 5.1
```

Comment:
nit: since you're using this feature remove/replace the empty line

Reply:
Thanks, I've pushed a new version fixing that (and also replacing 5.1 by 5.2).

Regarding FUSE, a quick look to the changelog shows that support for copy_file_range was added in version 3.4.1 (2018-12-22).  I may *try* to spend some time on this in the next couple of weeks (focus on the 'try' :-) ), but I suspect it will take me some time until I have anything ready for merging.  Thus, my personal preference here would be to have the kernel client fixed first.